crates/typos-dict/assets/allowed.csv

```diff
@@ -12,6 +12,7 @@ pervious,adjective
 perviously,adverb
 referer,http header field
 simulative,adjective
+spawnve,Microsoft-specific function (see https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/spawnve)
```

Comment:
I'm assuming, through bindings, this appears in enough programming languages that this is a generally allowed word, rather than a file-type specific one?